streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -146,5 +149,17 @@ public DebeziumArgs() {
 		@Parameter(names = { "--fhirDebeziumConfigPath" }, description = "Google cloud FHIR store")
 		public String fhirDebeziumConfigPath = "utils/dbz_event_to_fhir_config.json";
 		
+		@Parameter(names = { "--jdbcDriverClass" }, description = "JDBC MySQL driver class")
+		public String jdbcDriverClass = "com.mysql.cj.jdbc.Driver";
```

Comment:
@gitcliff  , since we already want to move dbz specific setting to a json config file , why not move these too see issue #51